### DISCUSSION TITLE
open outpaint support

### DISCUSSION THREAD

**Discussion by hoodady** [Panchovix/stable-diffusion-webui-reForge#15] [General]

**Comment by Panchovix** [comment]
Does it now work out of the box?

**Reply by hoodady** [reply]
yes thank you

**Reply by Lesteriax** [reply]
But it does NOT allow for use of controlnet as extras somehow, maybe it thinks it is not installed because it is integrated?

![image](https://github.com/user-attachments/assets/ae9e9949-a8c7-445b-986d-e71237b0a412)
